design/FY2022/enhacement_to_resilience_metric.md

```diff
@@ -0,0 +1,211 @@
+|-----------------------|---------------------------------|---------------------------------------|------------------|
+|Space<sub>1</sub>||||
+|...||||
+|Space<sub>N</sub>||||
+|Min||||
+|Max||||
+|Average||||
+|Sum||||
+
+Reporting period: Jun-01 to Jun-03
+<p style="text-align: center;"> Table 3. Sample report table of unmet degree hours (UDH).</p>
+
+**Discomfort-weighted Exceedance Hours**
+
+|	Very-cold Exceedance OccupantHours [h] | Cool Exceedance OccupantHours [h] | Warm Exceedance OccupantHours [h] | Very-hot Exceedance OccupantHours [h]
+|-----------------------|---------------------------------|---------------------------------------|------------------|
+|Space<sub>1</sub>||||
+|...||||
+|Space<sub>N</sub>||||
+|Min||||
+|Max||||
+|Average||||
+|Sum||||
+
+Reporting period: Jun-01 to Jun-03
+<p style="text-align: center;"> Table 4. Sample report table of discomfort-weighted exceedance hours.</p>
+
+When *ThermalResilienceSummary* is declared in *Output:Table:SummaryReports*, the above three tables will be generated and presented in the tabular reports. In particular, the *Hours of Safety* table will only be presented if an *Output:Table:ReportPeriod* object is defined for ThermalResilienceSummary report. 
+
+## Testing/Validation/Data Source(s): ##
+
+Two example files (the DOE reference small office and the one zone uncontrolled model) will be modified to demonstrate the use of the new feature. Simulation results will be manually checked/benchmarked using an excel spreadsheet with input and output from EnergyPlus runs.
+
+## IDD Object changes: ##
+
+Two extra fields will be added to the current People object to define the safe temperature thresholds for a certain kind of population.
+
+    People,
+      N7, \field Cold Stress Temperature Threshold
+          \type real
+          \units C
+          \note this is the indoor safe temperature threshold for cold stress
+    \default 15.56
+      N8, \field Heat Stress Temperature Threshold
+          \type real
+          \units C
+          \note this is the indoor safe temperature threshold for heat stress
+          \default 30
+
+We will add a new object *Output:Table:ReportPeriod* to add supplementary information to define the reporting period of certain summary tables.
+
+    Output:Table:ReportPeriod,
+        \memo This object allows the user to add a reporting period to a certain tabular report when the reporting period is not the whole simulation period. When defined, the declared summary report is based on aggregations from the start date to the end date defined. 
+        \memo Multiple run periods may be input.
+      A1, \field Name,
+          \required-field
+          \note descriptive name
+          \note cannot be blank and must be unique
+      A2, \field Report Name,
+          \key ThermalResilienceSummary
+          \key CO2ResilienceSummary
+          \key VisualResilienceSummary
+          \key …
```

Comment:
Perhaps add an option for "AllResilienceSummaries"

Reply:
Thanks, just added this key :)